plutus-contract/src/Language/Plutus/Contract/Effects/UtxoAt.hs

```diff
@@ -0,0 +1,66 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE ConstraintKinds     #-}
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE DeriveAnyClass      #-}
+{-# LANGUAGE DeriveGeneric       #-}
+{-# LANGUAGE DerivingStrategies  #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE MonoLocalBinds      #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE OverloadedLabels    #-}
+{-# LANGUAGE TypeApplications    #-}
+{-# LANGUAGE TypeOperators       #-}
+module Language.Plutus.Contract.Effects.UtxoAt where
+
+import           Data.Aeson                       (FromJSON, ToJSON)
+import           Data.Map                         (Map)
+import qualified Data.Map                         as Map
+import           Data.Row
+import           Data.Set                         (Set)
+import qualified Data.Set                         as Set
+import           GHC.Generics                     (Generic)
+import           Ledger                           (Address)
+import           Ledger.AddressMap                (AddressMap)
+import qualified Ledger.AddressMap                as AM
+import           Ledger.Tx                        (TxOut, TxOutRef)
+
+import           Language.Plutus.Contract.Request (Contract, ContractRow, requestMaybe)
+import           Language.Plutus.Contract.Schema  (Event (..), Handlers (..), Input, Output)
+
+type HasUtxoAt s =
+    ( HasType "utxo-at" UtxoAtAddress (Input s)
```

Comment:
Can we define these symbols as `type UtxoAt :: Symbol = "utxo-at"` or something? (I haven't tried)

Reply:
Yes it should work but the benefit is very small (you are saving 3 characters with every use, compared to an extra line for the type synonym). And if there's a typo anywhere the type checker will complain anyway.

Reply:
I was thinking more that it hides the perhaps-unfamiliar symbol stuff from people, but maybe that doesn't matter much.

Reply:
It's usually hidden in the `HasUtxoAt` constraint and the `UtxoAt` type, the only place where you need to write your own symbols is when you define endpoints.

But I made the change anyway :)

Reply:
We should do it for the others too, then, but we can do that another time.